beater/api/mux.go

```diff
@@ -220,8 +220,9 @@ func rumMiddleware(cfg *config.Config, _ *authorization.Handler, m map[request.R
 func sourcemapMiddleware(cfg *config.Config, auth *authorization.Handler) []middleware.Middleware {
 	msg := "Sourcemap upload endpoint is disabled. " +
 		"Configure the `apm-server.rum` section in apm-server.yml to enable sourcemap uploads. " +
-		"If you are not using the RUM agent, you can safely ignore this error."
-	enabled := cfg.RumConfig.IsEnabled() && cfg.RumConfig.SourceMapping.IsEnabled()
+		"If you are not using the RUM agent, you can safely ignore this error. " +
+		"If you are running APM Server managed by Fleet, you need to upload Sourcemaps directly to Elasticsearch."
```

Comment:
We know whether or not APM Server is managed by Fleet - can we change the error message based on that knowledge? Saves the user trying to figure out whether source maps are explicitly disabled, or implicitly because data streams are enabled.

Reply:
I was just being lazy... you got me